.circleci/config.yml

```diff
@@ -340,47 +348,73 @@ workflows:
   version: 2
   test:
     jobs:
+      - check_if_rebase_needed:
+          filters: # all branches, and /^v../ tags for build-publish...
+            tags:
+              only: /^v.*/
       - go-sqlite:
+          requires:
+            - check_if_rebase_needed
```

Comment:
Why does the rebase test stop other tests from running on failure?

Reply:
I was thinking that the tests would be stale, so there might not be any point in running them if you havent rebased. Would you rather them run anyway?

Reply:
Reverted this change, i see the value in it now